src/handlers/organizationsHandlers.ts

```diff
@@ -66,8 +76,56 @@ const getOrganizations = (
 		});
 };
 
+/** Takes an array of field values and returns an array of one best value per base field */
+export const extractGold = (
```

Comment:
@slifty Would you expect all the gold data extraction logic to be in the postgres function instead of here?

Reply:
Done in 408853eace888e19af3e7172377ea1d425830d43